fili-core/src/main/java/com/yahoo/bard/webservice/web/filters/BardLoggingFilter.java

```diff
@@ -83,6 +84,7 @@
     @Override
     public void filter(ContainerRequestContext request) throws IOException {
 
+        RequestLog.addIdPrefix(request.getHeaders().getFirst(X_REQUEST_ID_HEADER));
```

Comment:
What are allowed chars for this header? Are we going to use something like heroku's definition? https://devcenter.heroku.com/articles/http-request-id

I'll go with that for now.